doc/features/mapper/limitations-and-faq/README.md

```diff
@@ -0,0 +1,56 @@
+# Limitations and Frequently Asked Questions
+
+## Limitations
+
+There are some limitations in the Mapper design:
+
+- Secondary indexes and SOLR queries are not supported.
+- Case sensitive CQL identifiers (enclosed with double quotation marks) are not supported.
+- When mapping a model to multiple tables/views
+    - Columns with the same name must be of the same type.
+    - Updating primary keys of any of the tables is not supported and will result in additional rows being created on
+     the server side.
+- The following CQL features are not supported
+    - Deleting an individual map key/value pair is not supported, as in `DELETE favs['author'] FROM ...`
+    - Updating single UDT fields map key/values is not supported, e.g: `id.field = 3`
+
+Note that some limitations can be overcome by using [Custom Queries](../queries/#custom-queries).
+
+## FAQ
+
+### Should I specify the keyspace per mapping?
+
+If you are using a single keyspace containing all your data, its recommended that you set the keyspace name when 
+creating the `Client` instance. For example:
+
+```javascript
+const client = new Client({ contactPoints, keyspace: 'my_keyspace' });
+const mapper = new Mapper(client);
```

Comment:
I noticed that passing in a listing of models is not really required when inserting/updating/deleting data, but it seems to be needed when retrieving data.  For example, using the killrvideo 'videos' table:

```javascript
const mapper = new Mapper(client);
const videos = mapper.forModel('videos');
const p = videos.findAll();
```

I get the following error when attempting to retrieve data:

```
(node:17951) UnhandledPromiseRejectionWarning: TypeError: this._columns.get is not a function
    at ModelMappingInfo.getPropertyName (/home/tolbertam/Documents/Projects/nodejs-driver/lib/mapping/model-mapping-info.js:39:36)
    at columns.map.c (/home/tolbertam/Documents/Projects/nodejs-driver/lib/mapping/result-mapper.js:26:52)
    at Array.map (<anonymous>)
    at Function.getSelectAdapter (/home/tolbertam/Documents/Projects/nodejs-driver/lib/mapping/result-mapper.js:26:27)
    at _client.execute.then.rs (/home/tolbertam/Documents/Projects/nodejs-driver/lib/mapping/mapping-handler.js:124:50)
    at process._tickCallback (internal/process/next_tick.js:68:7)
```

I think there is at la bug here because it appears that `ModelMappingInfo.createDefault` passes in an empty array instead of an empty map.  If I change it from `utils.emptyArray` to `new Map()` everything seems to work.

Reply:
Awesome finding!